packages/react-core/src/components/Slider/Slider.tsx

```diff
@@ -19,6 +19,13 @@ export interface SliderStepObject {
   value: number;
 }
 
+export type SliderOnChangeEvent =
```

Comment:
I think this is ok... I don't know that is necessary. Would we want to upstate other event type definitions elsewhere to be the same for consistency?   What ever we decide, I would like to revisit the slider docs once this org [issue](https://github.com/patternfly/patternfly-org/issues/3423) gets resolve.  I don't love the hand generated  "Types" section in the docs.